zb-db/src/test/java/io/camunda/zeebe/db/impl/DbTenantAwareKeyColumnFamilyTest.java

```diff
@@ -136,4 +136,157 @@ void shouldDeleteValue() {
 
     assertThat(zbString).isNull();
   }
+
+  @Test
+  void shouldUseForeachValue() {
+    // given
+    upsertKeyValuePair(123L, "foo", "tenantId");
+    upsertKeyValuePair(124L, "bar", "tenantId");
+    upsertKeyValuePair(125L, "baz", "otherTenantId");
+    upsertKeyValuePair(777L, "jackpot", "tenantId");
+    upsertKeyValuePair(888L, "lastOne", "otherTenantId");
+
+    // when
+    final var values = new ArrayList<>();
+    columnFamily.forEach((value) -> values.add(value.toString()));
+
+    // then
+    assertThat(values).containsExactly("foo", "bar", "baz", "jackpot", "lastOne");
+  }
```

Comment:
🔧 It would be easier to grasp if the tenant and the value were swapped as parameters of the `upsertKeyValuePair` method because the tenant id is part of the key that points to the value